docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+# Configuration for the services that are required to run an analysis of the
+# data from riff edu mattermost.
+# see https://www.metabase.com/docs/latest/operations-guide/running-metabase-on-docker.html
+version: '3.6'
+services:
+
+  target-db:
+    image: mysql:5.7
+    container_name: mysql-business-analytics
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+    ports:
+      - '3306:3306'
+    volumes:
+      - target-db-data:/var/lib/mysql
+
+  metabase:
```

Comment:
Going along w/ the above comment about service names, perhaps `ba-metabase`. Having a common prefix for the services defined in a compose file that should all work together is helpful in identifying them in the lists of images/containers.

Reply:
Added consistent names to the services.